docs/Storage.md

```diff
@@ -51,7 +51,8 @@ storage1 = manager.create_storage(
     zone='fi-hel1',
     size=10,
     tier="maxiops",
-    title="my storage disk"
+    title="my storage disk",
+    encrypted=True
```

Comment:
Maybe shouldn't suggest enabling encryption by default in examples as it can slow down things and user might be confused why 🤔 

```suggestion
    encrypted=False
```